ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -63,20 +63,9 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.10 and @aws-ipi'
       UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @aws-ipi'
-    post:
-    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
-    pre:
-    - chain: cucushift-installer-rehearse-aws-ipi-provision
     test:
-    - ref: openshift-extended-upgrade-pre
-    - ref: cucushift-pre
-    - ref: cucushift-upgrade-prepare
-    - ref: cucushift-upgrade-prehealthcheck
-    - ref: cucushift-upgrade-toimage
-    - ref: cucushift-upgrade-healthcheck
-    - ref: openshift-extended-upgrade-post
-    - ref: cucushift-upgrade-check
-    - ref: cucushift-upgrade-report
+    - chain: openshift-upgrade-test
```

Comment:
This chain `openshift-upgrade-test` seems for golang+cucushift cases. And there is another upgrade chain `https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/cucushift-upgrade-chain.yaml` for cucushift only. I wonder if we need both of them? if not, maybe we need integrate them into one. if yes, we hold them into two dirs by purpose?

Reply:
Once we finished migrating existing upgrade jobs, we can remove cucushift-upgrade-chain

Reply:
ack